### DISCUSSION TITLE
docker-compose stack, reboot, container network fails to start some containers

### DISCUSSION THREAD

**Discussion by jabajools** [qdm12/gluetun#975] [Q&A]
Evening all,

I'm currently running a compose file that lists Gluetun and then a number of other services that use Gluetun as their network. 

On initially bringing the compose file up, everything works perfectly. However, when the system reboots, only those services in the stack which do not use Gluetun as their network start, Gluetun and those containers dependent on it do not start unless I manually drop and then up the stack once more.

This happens despite Gluetun and all other services having restart unless stopped or restart always defined.

Sadly there are no logs as those containers just don't start?

**Comment by qdm12** [comment]
Are all the gluetun connected containers in the same docker-compose.yml as Gluetun? Maybe use `network_mode: "service:gluetun"` to indicate the dependency

I guess they fail since gluetun isn't up so they can't start 🤔
